app/helpers/application_helper.rb

```diff
@@ -16,4 +16,17 @@ def facet_has_any_selected_values?(key)
   def absolute_url_for(path)
     URI.join(Plek.current.website_root, path)
   end
+
+  def page_metadata_links(metadata)
+    metadata.inject({}) do |memo, (type, links)|
+      memo[type] = arr_to_links(links)
+      memo
```

Comment:
It's more idiomatic to use `merge` rather than mutation when using `inject`/`reduce`:

```ruby
def page_metadata_links(metadata)
  metadata.reduce({}) { |memo, (type, links)|
    memo.merge(type => arr_to_links(links))
  }
end
```

If you think mutation would be a better approach (I don't!), then `with_object` lets you use mutation rather than return values:

```ruby
def page_metadata_links(metadata)
  metadata.each.with_object({}) do |(type, links), memo|
    memo[type] = arr_to_links(links)
  end
end
```